src/decaylanguage/dec/dec.py

```diff
@@ -655,21 +660,37 @@ def list_decay_modes(self, mother, pdg_name=False):
             for mode in self._find_decay_modes(mother)
         ]
 
-    def _decay_mode_details(self, decay_mode):
+    def _decay_mode_details(self, decay_mode, display_photos_keyword=False):
         """
         Parse a decay mode (Tree instance)
         and return the relevant bits of information in it.
+
+        Parameters
+        ----------
+        decay_mode: str
+            Input decay mode to list its details.
+        display_photos_keyword: boolean, optional, default=False
+            Don't omit the "PHOTOS" keyword in decay models.
         """
 
         bf = get_branching_fraction(decay_mode)
         fsp_names = get_final_state_particle_names(decay_mode)
         model = get_model_name(decay_mode)
         model_params = get_model_parameters(decay_mode)
 
+        if display_photos_keyword and list(decay_mode.find_data("photos")):
+            model = "PHOTOS " + model
+
         return (bf, fsp_names, model, model_params)
 
     def print_decay_modes(
-        self, mother, pdg_name=False, print_model=True, ascending=False
+        self,
+        mother,
+        pdg_name=False,
+        print_model=True,
```

Comment:
I'm setting the default to True since it seems reasonable to have the info printed by default (I had not realised the interest of this before your PR ;-)).